test/lib/Streamly/Test/Prelude/Common.hs

```diff
@@ -1180,7 +1181,7 @@ transformCombineOpsCommon constr desc eq t = do
 
     -- reordering
     prop (desc <> " reverse") $ transform reverse t S.reverse
-    -- prop (desc <> " reverse'") $ transform reverse t S.reverse'
+    prop (desc <> " reverse'") $ transform reverse t S.reverse'
```

Comment:
This only works on ordered streams, so I don't think it should be in `transformCombineOpsCommon`

Reply:
How is it different than `reverse`? They both should be moved?

Reply:
Yes both of them should be moved to the Ordered ops I think

Reply:
I have too much stuff to do, I will just add a comment and merge it for now.